MATLAB/DFA/src/EDA/exper_data.m

```diff
@@ -0,0 +1,360 @@
+T_avg_TU8902_NT = Tabl_average_TU8902(:, {'metabolite', 'TU8902_shNT_no_dox_1','TU8902_shNT_2d_dox_1', 'TU8902_shNT_4d_dox_1', 'TU8902_shNT_no_dox_media_1', 'TU8902_shNT_2d_dox_media_1', 'TU8902_shNT_4d_dox_media_1'});
+
+T_std_TU8902_GOT1 = Tabl_std_TU8902(:, {'metabolite', 'TU8902_shGOT1_no_dox_1','TU8902_shGOT1_2d_dox_1', 'TU8902_shGOT1_4d_dox_1', 'TU8902_shGOT1_no_dox_media_1', 'TU8902_shGOT1_2d_dox_media_1', 'TU8902_shGOT1_4d_dox_media_1'});
+T_std_TU8902_NT = Tabl_std_TU8902(:, {'metabolite', 'TU8902_shNT_no_dox_1','TU8902_shNT_2d_dox_1', 'TU8902_shNT_4d_dox_1', 'TU8902_shNT_no_dox_media_1', 'TU8902_shNT_2d_dox_media_1', 'TU8902_shNT_4d_dox_media_1'});
+
+%create numeric arrays
+avg_TU8902_GOT1 = table2array(T_avg_TU8902_GOT1(:, 2:end));
+avg_TU8902_NT = table2array(T_avg_TU8902_NT(:, 2:end));
+
+std_TU8902_GOT1 = table2array(T_std_TU8902_GOT1(:, 2:end));
+std_TU8902_NT = table2array(T_std_TU8902_NT(:, 2:end));
+
+%% Create a corelation matrix and a following heatmap
+%Do this by first converting all the NaN values into 0 and then create a corelation matrix of the metabolites using an array of averages.  
+% create arrays from the tables above. 
+mean_metabolomic_array_TU8902 = table2array(Tabl_average_TU8902(:,2:end));
+std_metabolomic_array_TU8902 = table2array(Tabl_std_TU8902(:,2:end));
+
+mean_metabolomic_array_HCT116 = table2array(Tabl_average_HCT116(:,2:end));
+std_metabolomic_array_HCT116 = table2array(Tabl_std_HCT116(:,2:end));
+
+%Convert all the NaN values to 0
+mean_metabolomic_array_TU8902(isnan(mean_metabolomic_array_TU8902)) = 0;
+mean_metabolomic_array_HCT116(isnan(mean_metabolomic_array_HCT116)) = 0;
+
+%r matrix and its heatmap - for the 12 conditions
+averageCorrMat_TU8902 = corrcoef(mean_metabolomic_array_TU8902);
+figure (1);
+heatmap(all_conditions, all_conditions, averageCorrMat_TU8902);
+title 'Correlation Matrix of TU8902 Averages across 12 Conditions';
+
+averageCorrMat_HCT116 = corrcoef(mean_metabolomic_array_HCT116);
+figure (2);
+heatmap(all_conditions, all_conditions, averageCorrMat_HCT116);
+title 'Correlation Matrix of HCT116 mean values over 12 conditions';
+
+% create r matrix and heatmaps for the metabolomics
+corrmat = zeros(12);
+for i = 1:size(mean_metabolomic_array_TU8902, 1)
+    metabolite = mean_metabolomic_array_TU8902(i, :);
+    tmp = corr(metabolite);
+    corrmat(:, i) = diag(tmp);
+end 
+% NOTE: Running this gives a matrix of NaNs.
+
+%% Create the correlation matrix, CV 
+%CV matrix
+CV_TU8902 = std_metabolomic_array_TU8902 ./ mean_metabolomic_array_TU8902;
+CV_TU8902_GOT1 = std_TU8902_GOT1 ./ avg_TU8902_GOT1;
+CV_TU8902_NT = std_TU8902_NT ./ avg_TU8902_NT;
+
+CV_HCT116 = std_metabolomic_array_HCT116 ./ mean_metabolomic_array_HCT116;
+CV_HCT116_GOT1 = std_HCT116_GOT1 ./ avg_HCT116_GOT1;
+CV_HCT116_NT = std_HCT116_NT ./ avg_HCT116_NT;
+
+%% Create heatmaps to visualize the averages
+figure (3);
+heatmap(all_conditions, metabolomics(:,1),mean_metabolomic_array_TU8902, 'ColorScaling', 'log');

```

Comment:
It may be easier to interpret after you do the NaN / CV filter. You don't have to visualize all datasets, just the one that contains ~60-70% of the dataset